docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -0,0 +1,182 @@
+---
+approvers:
+- smarterclayton
+title: Using a KMS provider for data encryption
+---
+{% capture overview %}
+This page shows how to configure a Key Management Service (KMS) provider and plugin to enable secret data encryption.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include task-tutorial-prereqs.md %}
+
+* Kubernetes version 1.10.0 or later is required
+
+* etcd v3 or later is required
+
+*The KMS provider is alpha in Kubernetes version 1.10.0, which means that it may change without notice. You may be required to decrypt your data prior to upgrading to 1.11.0
```

Comment:
In general, do not put statement about future here. We don't know what will happen in v1.11 yet.

Reply:
Also, the `*` sign in line 18 is treated as a flag for slant.

So ... please consider remove line 18 completely.

Reply:
I took this line from https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
Removed